projboard/tests/test_client.py

```diff
@@ -5,3 +5,9 @@
 def test_homepage(client):
     response = client.get("/")
     assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_create_article(client):
```

Comment:
Please thing how to create user from ui
And then verify it is created

Please take a look here 
https://pytest-django.readthedocs.io/en/latest/helpers.html#client-django-test-client

It uses this class also [django.test.Client](https://docs.djangoproject.com/en/stable/topics/testing/tools/#django.test.Client).

see how to create post request

Reply:
When you write

>  Please thing how to create user from ui

you mean to User authentication ?

Reply:
Added test for post request at [8f0ea9e](https://github.com/redhat-beyond/Smarticle/pull/70/commits/8f0ea9e51f38c4edcc21b4569fa50442915f2d40)